source/stylesheets/main.scss

```diff
@@ -229,19 +229,19 @@ select:hover, select:focus, .mak:hover, .mak:focus {
 		svg[opened-eye], svg[crossed-eye] {
 			position: absolute;
 			@include transition(opacity $transition_duration);
-			fill: $def-darkgrey;
+			fill: #757575;
```

Comment:
Won't work: the selectors are messed up. `opened-eye` and `crossed-eye` are not attributes on svg. You can access these svg elements via `.opened-eye svg` and `.crossed-eye svg`, so using the parent spans' CSS classes. Remember to use the classnames suggested above.